speaker/main.go

```diff
@@ -352,6 +367,11 @@ func (c *controller) SetConfig(l log.Logger, cfg *config.Config) k8s.SyncState {
 		return k8s.SyncStateError
 	}
 
+	if !c.bgpImplementationSupportAggrLenV6Cfg(cfg.Pools) {
+		level.Error(l).Log("op", "setConfig", "error", "unsupported configuration", "msg", "configuration is missing, MetalLB will not function")
```

Comment:
Can you extend the error with the reason? It will be easier for the users to self debug.
Also, I'd reword this to "configuration is ignored"